tensorrt_llm/_torch/peft/lora/loaders.py

```
@@ -0,0 +1,439 @@
+    """
+    with tarfile.open(nemo_archive_path) as tar:
+        try:
+            model_weights_file = tar.extractfile("model_weights.ckpt")
+            model_config_file = tar.extractfile("model_config.yaml")
+        except KeyError:
+            try:
+                model_weights_file = tar.extractfile("./model_weights.ckpt")
+                model_config_file = tar.extractfile("./model_config.yaml")
+            except KeyError:
+                err_str = "Both model_weights paths not found in the tar archive."
+                raise Exception(err_str)
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)